docker-compose.dev.yml

```diff
@@ -6,3 +6,28 @@ services:
       context: .
       dockerfile: Dockerfile.dev
     container_name: avocadoro-backend
+    depends_on:
+      - postgres
+    volumes:
+      - .:/app
+
+  postgres:
+    image: postgres:11-alpine
+    container_name: avocadoro-db
+    restart: "unless-stopped"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres123
```

Comment:
the credentials out there again

Reply:
This is for the mock database for dev, so there should not be any security concern.